modules/aws-ec2-instance/main.tf

```diff
@@ -36,11 +62,11 @@ resource "aws_network_interface" "tamr-instance-network" {
 }
 
 data "template_cloudinit_config" "bootstrap-scripts" {
-  count         = length(var.bootstrap_scripts)
+  count         = length(var.bootstrap_scripts) + 1
```

Comment:
To me this reads as its looping over the scripts twice. We are creating a data source N times. I think that bug existed before though.

Reply:
Yeah, I think you are right. The count can be removed.